<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeWidget</class>
 <widget class="QWidget" name="TreeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="3">
    <widget class="BookmarksWidget" name="Bookmarks" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="standardQuery">
        <property name="text">
         <string>Standard</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="advancedQuery">
        <property name="text">
         <string>Advanced Queries</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="pageSize">
        <property name="text">
         <string>ALL</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Page Size:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="bookmarksButton">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Bookmarks</string>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/icons/images/bookmarks2.png</normaloff>:/icons/images/bookmarks2.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="closeButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/icons/images/reloadpage.png</normaloff>:/icons/images/reloadpage.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="TreePanel" name="tree">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="animated">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>                                Tree                                                     </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="selection">
     <property name="title">
      <string>Queries</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="configNameRadioButton">
        <property name="text">
         <string>SimCond/ProcessingPass/Eventtype/Production/FileType/Program/Files</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Event type/SimCond/ProcessingPass/Production/FileType/Program/Files</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="productionRadioButton">
        <property name="text">
         <string>Production lookup</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="runLookup">
        <property name="text">
         <string>Run lookup</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="configNameRadioButton">
   <property name="text">
    <string>config_click</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BookmarksWidget</class>
   <extends>QWidget</extends>
   <header>LHCbDIRAC/NewBookkeepingSystem/Gui/Widget/BookmarksWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TreePanel</class>
   <extends>QTreeWidget</extends>
   <header>LHCbDIRAC/NewBookkeepingSystem/Gui/Widget/TreePanel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
